### DISCUSSION TITLE
Text to Speech

### DISCUSSION THREAD

**Discussion by imperatore-spqr** [deepgram/community#346] [General help] [answered]
Hello, I only want to use Deep gram for its Text to Spech part but I cannot find any documentation for setup ca

I only see documentation for transcription, but I'm not interested

Yet here, they say that they have a Text to speech feature but I can't find it:
https://deepgram.com/learn/best-speech-to-text-apis

**Comment by jjmaldonis** [comment, marked as answer]
Hey @imperatore-spqr, currently Deepgram provides functionality for [speech-to-text](https://deepgram.com/product/transcription) and [speech understanding features](https://deepgram.com/product/speech-understanding) such as summarization, topic detection, etc. Text-to-speech is not currently available.

**Comment by natalierutgers** [comment]
Hi @imperatore-spqr what are you hoping to use text to speech for? Are you already using another vendor?
